src/scatter_utilities/estimate_scatter.cxx

```diff
@@ -33,25 +33,103 @@
   of the parameter file.
 */
 
-#include "stir/scatter/ScatterEstimationByBin.h"
+#include "stir/scatter/ScatterEstimation.h"
 #include "stir/Succeeded.h"
 /***********************************************************/     
 
+static void print_usage_and_exit()
+{
+    std::cerr<<"This executable runs a Scatter simulation method based on the options "
+               "in a parameter file";
+    std::cerr<<"\nUsage:\n simulate_scatter scatter_simulation.par\n";
+    std::cerr<<"Example parameter file:\n\n"
+               "Scatter Estimation Parameters :=\n"
+               ";Run in debug mode\n"
+               ";; A new folder called extras will be created, in which many\n"
+               ";; extra files will be stored \n"
+               "run in debug mode := 1\n"
+              " recompute initial activity image := 1\n"
+               "initial activity image filename := \n"
+
+               "zoom xy := 0.3\n"
+               "zoom z := 1.0\n"
+               "input file := \n"
+               "recompute attenuation projdata := 0\n"
+               "αttenuation projdata filename := \n"
+               "attenuation image filename := \n"
+
+               ";Upsample and fit \n"
+               "; defaults to 3.\n"
+               "half filter width := \n"
+               "remove interleaving := \n"
+
+               "; If provided it will be given to the ScatterSimulation \n"
+               "; subsampled attenuation imagw filename := \n"
+
+               "maximum scale value := \n"
+               "minimum scale value := \n"
+
+               "normalisation coefficients filename := \n"
+               "recompute mask projdata := \n "
+               "mask projdata filename := \n "
+
+               "recompute mask image :=\n "
+               "mask image filename := \n"
+
+               "mask image prostfilter filename := "
+               "mask image max threshold := "
+               "mask image additive scalar := "
+               "mask image min threshold := "
+               "mask image times scalar :="
+               "; End of Mask\n"
+
+               "tail fitting par filename :=\n "
+
+               ";Backgroud data\n"
```

Comment:
misspelling here and below. I presume same problems as in the example .par file

Reply:
:ok: